index.js

```diff
@@ -12,15 +12,16 @@ var net = require("net"),
     parsers = [], commands,
     connection_id = 0,
     default_port = 6379,
-    default_host = "127.0.0.1";
-
-// can set this to true to enable for all connections
-exports.debug_mode = false;
+    default_host = "127.0.0.1",
+    debugModeEnabled = /\bredis\b/i,
+    debug = function () {
+        if (debugModeEnabled.test(process.env.NODE_DEBUG)) {
```

Comment:
Good point and done. Now it's possible to redirect the call to console.error without having to convert the arguments.